src/_pyinstaller_hooks_contrib/hooks/stdhooks/hook-mecab.py

```diff
@@ -0,0 +1,16 @@
+# ------------------------------------------------------------------
+# Copyright (c) 2023 PyInstaller Development Team.
+#
+# This file is distributed under the terms of the GNU General Public
+# License (version 2.0 or later).
+#
+# The full license is available in LICENSE.GPL.txt, distributed with
+# this software.
+#
+# SPDX-License-Identifier: GPL-2.0-or-later
+# ------------------------------------------------------------------
+
+from PyInstaller.utils.hooks import collect_data_files
+
+datas = collect_data_files('mecab')
+datas = collect_data_files('mecab_ko_dic')
```

Comment:
This should be `datas += collect_data_files('mecab_ko_dic')`, otherwise you lose the `mecabrc` file from `mecab` package that you collected in previous call.